docs/installing-solidity.rst

```diff
@@ -154,23 +154,32 @@ Gentoo Linux also provides a solidity package that can be installed using ``emer
 Building from Source
 ====================
 
-Prerequisites - Linux
----------------------
+Prerequisites - All Operating Systems
+-------------------------------------
 
-You need to install the following dependencies for Linux builds of Solidity:
+The following are dependencies for all builds of Solidity:
 
 +-----------------------------------+-------------------------------------------------------+
 | Software                          | Notes                                                 |
 +===================================+=======================================================+
-| `Git for Linux`_                  | Command-line tool for retrieving source from Github.  |
+| `CMake`_                          | Cross-platform build file generator.                  |
++-----------------------------------+-------------------------------------------------------+
+| `Boost`_                          | C++ libraries.                                        |
++-----------------------------------+-------------------------------------------------------+
+| `Git`_                            | Command-line tool for retrieving source code.         |
++-----------------------------------+-------------------------------------------------------+
+| `z3`_ (Optional)                  | For use with SMT checker.                             |
```

Comment:
Should also add cvc4 (Optional) - For use with SMT checker - http://cvc4.cs.stanford.edu/web/